### DISCUSSION TITLE
Why can't I find the download for stabilityai/stable-diffusion-xl-refiner-0.9?

### DISCUSSION THREAD

**Discussion by sevenlucas7** [vladmandic/sdnext#1757] [Q&A]
I can't find the download for stabilityai/stable-diffusion-xl-refiner-0.9 in the Model's tab.
What should I do?
<img width="1087" alt="image" src="https://github.com/vladmandic/automatic/assets/131904649/5b0206d0-3c71-46e5-bf38-477dbdb5b875">

**Comment by vladmandic** [comment]
read wiki

**Reply by sevenlucas7** [reply]
hello，
I have followed the instructions on the wiki, filled out the form, and copied the access token. I successfully downloaded the XL-BASE model. Then, when I tried to download the refiner by inputting "stabilityai/stable-diffusion-xl-refiner-0.9," there was no connection. Why is this happening?

**Reply by ETA-22** [reply]
Just download it manually from the website?
